src/components/button-share/facebook-icon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
Icons should live in the design system and be used like this:
`import ExpandMoreIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/ExpandMore.svg";`